### PR TITLE
Merge `next` into `master`

### PR DESCRIPTION
**Pull request by jsumners**
As titled.

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -31,17 +31,17 @@
   },
   "homepage": "https://github.com/fastify/fastify-bearer-auth#readme",
   "devDependencies": {
-    "@fastify/auth": "^4.0.0",
-    "@fastify/pre-commit": "^2.0.2",
-    "@types/node": "^20.1.0",
-    "fastify": "^4.0.0-rc.2",
-    "standard": "^17.0.0",
-    "tap": "^16.0.0",
+    "@fastify/auth": "^4.6.1",
```
Comment (jsumners) [inline package.json:34]: Blocking until this one is released.

File: package.json [hunk 2]
```diff
@@ -31,17 +31,17 @@
...
+    "@fastify/pre-commit": "^2.1.0",
+    "@types/node": "^20.12.7",
+    "fastify": "^4.26.2",
```
Comment (jsumners) [inline package.json:37]: Update.

File: package.json [hunk 3]
```diff
@@ -31,17 +31,17 @@
...
+    "standard": "^17.1.0",
+    "tap": "^18.7.2",
     "tsd": "^0.31.0"
   },
   "dependencies": {
     "@fastify/error": "^3.4.1",
-    "fastify-plugin": "^4.0.0"
+    "fastify-plugin": "^4.5.1"
```
Comment (jsumners) [inline package.json:44]: Update.

**Pull request merged**